src/args.rs

```diff
@@ -1,6 +1,6 @@
 use clap::{clap_app, crate_version, AppSettings};
 
-pub fn get() -> clap::App<'static, 'static> {
+pub fn args_get() -> clap::App<'static, 'static> {
```

Comment:
I personally prefer leaving it as `get()`, since that along with not reexporting the modules will make the function call look like `args::get()` instead of `args::args_get()` which is cleaner to me.